### DISCUSSION TITLE
Diplicate Transaction

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#3901] [Q&A] [answered]
If I copy a transaction by STRG-D the the date is either today's or the date of the last posting.
This depends on the settings in the options.

How does it work that the date of the original booking is retained?

**Comment by vomikan** [comment, marked as answer]
I don't understand the question.

**Comment by TomAff** [comment]
I have these two postings. I press STRG-D to make a copy of the one at 15.10.21
![2021-10-02 21_53_38-MMEX Options](https://user-images.githubusercontent.com/46572503/135730237-ae972cd2-e775-4692-b87d-799a325e3754.jpg)

If option is
![2021-10-02 21_50_59-MMEX Options](https://user-images.githubusercontent.com/46572503/135730143-55af9730-689f-43e6-b3a0-c5ffaefa141f.jpg)

The suggestion of the date is today
![2021-10-02 21_52_16-Duplicate Transaction](https://user-images.githubusercontent.com/46572503/135730137-a0241681-aeed-4eb0-8885-c64f5687b9e7.jpg)

If option is this
![2021-10-02 21_53_38-MMEX Options](https://user-images.githubusercontent.com/46572503/135730174-94e3ce63-6f86-4b45-83a8-f523753ca1eb.jpg)

Suggestion is
![2021-10-02 21_53_38-MMEX Options](https://user-images.githubusercontent.com/46572503/135730201-30807941-1d2a-43b2-ac76-aeddd49376e3.jpg)

I would like to have the date of the copied transaction as suggestion.
In this case 15.10. I think it was so in an older version?!
Is it possible in 1.5.8? Or is it another shortcut?

If I use STRG-C STRG-V. Date of the new transaction is today

**Comment by tactilis** [comment]
See https://github.com/moneymanagerex/moneymanagerex/issues/3784

**Comment by TomAff** [comment]
Oh. Sorry, didn't see this.

But now I've seen, that the german translation of these two points is the same.
![2021-10-03 11_39_00-MMEX Optionen](https://user-images.githubusercontent.com/46572503/135748161-221cf410-31e6-4f74-9fa4-0e916bc62ee9.jpg)

First one should be
Originaldatum beim Kopieren/Einfügen von Buchungen beibehalten. (think this is better)

second
Originaldatum beim Duplizieren von Buchungen beibehalten.
